resources/images/s3/filestorage.go

```diff
@@ -54,10 +54,21 @@ func NewSimpleStorageServiceStatic(bucket, key, secret, region, token, uri strin
 		config = config.WithDisableSSL(sslDisabled).WithEndpoint(uri)
 	}
 
+	config.S3ForcePathStyle = aws.Bool(true)
 	sess := session.New(config)
 
+	client := s3.New(sess)
+
+	// minio requires explicit bucket creation
+	cparams := &s3.CreateBucketInput{
+		Bucket: aws.String(bucket), // Required
+	}
+
+	// Create a new bucket using the CreateBucket call.
+	client.CreateBucket(cparams)
```

Comment:
I'm not sure about error handling here.
If we cannot create bucket we should probably exit (panic?).
If the bucket already exists it is probably ok;
maybe we should check if this bucket exists and try to create it only if not,
or maybe we shoud check if there is a way to setup minio server with predefined buckets